src/Plugin/Field/FieldType/MediaTrackItem.php

```diff
@@ -202,14 +202,26 @@ public static function generateSampleValue(FieldDefinitionInterface $field_defin
     // $sample_file_text += randomText()
     // }
     // $file = file_write($sample_file_text, 'random_filename.vtt');
+    $sample_file_text = <<<EOT
+WEBVTT
+
+00:00:00.500 --> 00:00:02.000
+This is a test subtitle.
+
+00:00:04.500 --> 00:00:06.300
+This is another test subtitle.
+EOT;
+
+    $file = file_write($sample_file_text, 'sample_file.vtt');
```

Comment:
`file_write()` doesn't appear to exist? Thinking, should probably be something like:

```suggestion
    $file_repo = \Drupal::service('file.repository');
    $file = $file_repo->writeData($sample_file_text, "public://sample_file.vtt");
```